bitnami/etcd/README.md

```diff
@@ -83,10 +83,16 @@ The following tables lists the configurable parameters of the etcd chart and the
 | `auth.client.useAutoTLS`                        | Switch to automatically create the TLS certificates                                                                                                       | `false`                                                     |
 | `auth.client.enableAuthentication`              | Switch to enable host authentication using TLS certificates. Requires existing secret.                                                                    | `secret`                                                    |
 | `auth.client.existingSecret`                    | Name of the existing secret containing cert files for client communication.                                                                               | `nil`                                                       |
+| `auth.client.certFilename`                        | Name of the file containing the client certificate.                                                                                                     | `cert.pem`                                                  |
+| `auth.client.certKeyFilename`                     | Name of the file containing the client certificate private key.                                                                                         | `key.pem`                                                   |
+| `auth.client.caFilename`                          | Name of the file containing the client CA certificate. If not specified and `enableAuthentication: true` or `rbac.enabled: true`, the default is is `ca.crt`. | `""`                                                  |
```

Comment:
Please format these line properly as the rest of the table

Reply:
Fixed